app/tests/test_config.py

```diff
@@ -59,6 +61,47 @@ def test_local_env_vars_config_initialized(monkeypatch):
     assert config.FLASKS3_BUCKET_NAME == "test_flasks3_bucket_name"
 
 
+def test_local_env_config_variable_not_set_error(monkeypatch):
+    """
+    GIVEN an environment variable 'DEFAULT_DATE_FORMAT' is not set in local .env file
+    WHEN Config is initialized
+    THEN it should raise an exception with error
+    """
+    monkeypatch.setenv(
+        "SQLALCHEMY_DATABASE_URI", "test_sqlalchemy_database_uri"
+    )
+    monkeypatch.setenv("DB_PORT", "5432")
+    monkeypatch.setenv("DB_HOST", "test_db_host")
+    monkeypatch.setenv("DB_USER", "test_db_user")
+    monkeypatch.setenv("DB_PASSWORD", "test_db_password")
+    monkeypatch.setenv("DB_NAME", "test_db_name")
+    monkeypatch.setenv("KEYCLOAK_BASE_URI", "test_keycloak_base_uri")
+    monkeypatch.setenv("KEYCLOAK_CLIENT_ID", "test_keycloak_client_id")
+    monkeypatch.setenv("KEYCLOAK_REALM_NAME", "test_keycloak_realm_name")
+    monkeypatch.setenv(
+        "KEYCLOAK_CLIENT_SECRET", "test_keycloak_client_secret"
+    )  # pragma: allowlist secret
+    monkeypatch.setenv(
+        "SECRET_KEY", "test_secret_key"  # pragma: allowlist secret
+    )
+    monkeypatch.setenv("DEFAULT_PAGE_SIZE", "test_default_page_size")
+    monkeypatch.setenv("RECORD_BUCKET_NAME", "test_record_bucket_name")
+    monkeypatch.setenv("AWS_REGION", "test_region")
+    monkeypatch.setenv("FLASKS3_ACTIVE", "False")
+    monkeypatch.setenv("FLASKS3_CDN_DOMAIN", "test_flasks3_cdn_domain")
+    monkeypatch.setenv("FLASKS3_BUCKET_NAME", "test_flasks3_bucket_name")
+
+    config = EnvConfig()
+
+    with pytest.raises(Exception) as error:
+        inspect.getmembers(config)
```

Comment:
so should we place this in app/_init__.py so we error out on startup?

Reply:
line inspect.getmembers(config_class()) has been added in app/init__.py as requested to check config var values exist or not